frmts/vrt/pixelfunctions.cpp

```diff
@@ -15,6 +15,12 @@
 #include "gdal.h"
 #include "vrtdataset.h"
 
+#define exprtk_disable_caseinsensitivity
+#define exprtk_disable_rtl_io
+#define exprtk_disable_rtl_io_file
+#define exprtk_disable_rtl_vecops
+#define exprtk_disable_string_capabilities
+#include <exprtk.hpp>
```

Comment:
Maybe the following to avoid namespace collisions in case GDAL is integrated with another software also using exprtk?
```suggestion
#define exprtk gdal_exprtk
#include <exprtk.hpp>
```